lms/resources/lti_launch.py

```diff
@@ -122,7 +122,7 @@ def resource_link_id(self):
         ):
             return resource_link_id
 
-        return self._request.POST.get("resource_link_id")
+        return self._request.lti_params.get("resource_link_id")
```

Comment:
This is a big change even if it looks a bit silly.

This runs from a predicate which run before "view derivers" (we add `parsed_params` to the request in a web deriver).

That means that self.request.params will contain `id_token` but we don't have access to it's containing information. Exposing `lti_params` directly on the request allows to fix this.